packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -295,6 +371,15 @@ const start = async (zcf, privateArgs) => {
   const creatorFacet = makeGovernorFacet(
     Far('AMM Fee Collector facet', {
       makeCollectFeesInvitation,
+      /**
+       * Must be called before adding pools. Not provided at contract start time due to cyclic dependency.
+       *
+       * @param {AssetReservePublicFacet} facet
+       */
+      setReserveDepositFacet: facet => {
+        // FIXME more than deposits
+        reserveDepositFacet = facet;
```

Comment:
I think this should assert `!reserveDepositFacet` before setting.

I'd prefer that this facet was visible via governance, but failing that, it should be set-once.